website/content/docs/connect/connect-internals.mdx

```diff
@@ -11,21 +11,20 @@ This topic describes how many of the core features of Consul's service mesh func
 It is not a prerequisite,
 but this information will help you understand how Consul service mesh behaves in more complex scenarios.
 
-Consul Connect is the component shipped with Consul that enables service mesh functionality. The terms _Consul Connect_ and _Consul service mesh_ are used interchangeably throughout this documentation.
```

Comment:
I think this is important context to retain because it clearly states how the term "Connect" relates to Consul service mesh.